modulefiles/build.gaea.intel

```diff
@@ -0,0 +1,22 @@
+#%Module#####################################################
+## Build module for Gaea
+#############################################################
+
+module switch intel/18.0.6.288
+
+module load git/2.26.0
+module load cmake/3.17.0
+
+# hpc-stack installed as a flat install at:
+# /ncrc/home2/Rahul.Mahajan/dev/opt
+
+setenv ESMFMKFILE /ncrc/home2/Rahul.Mahajan/dev/opt/lib/esmf.mk
+
+prepend-path PATH /ncrc/home2/Rahul.Mahajan/dev/opt/bin
+prepend-path CMAKE_PREFIX_PATH /ncrc/home2/Rahul.Mahajan/dev/opt
+prepend-path LD_LIBRARY_PATH /ncrc/home2/Rahul.Mahajan/dev/opt/lib
+prepend-path LD_LIBRARY_PATH /ncrc/home2/Rahul.Mahajan/dev/opt/lib64
+
```

Comment:
FYI, I am not responsible for maintaining this space.
This is provided as a proof of concept, that UFS utils can indeed be built on Gaea.

Reply:
Understood.

Reply:
I can install hpc-stack on gaea in the project directories that we have been using for the ufs-weather-model support thus far, if needed.